Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.h

```diff
@@ -85,6 +88,12 @@ namespace AZ
             /// false.
             bool AddSoftName(SoftNameSetting* newSoftname);
 
+            void PopulateDefaultSoftNameSettings();
+
+            template<class SoftNameSettingsType>
+            void AddSoftNameSettingsFromSettingResitry(
```

Comment:
typo: `AddSoftNameSettingsFromSettingResitry` is misspelling Registry
```suggestion
            void AddSoftNameSettingsFromSettingRegistry(
```